torchvision/transforms/functional_pil.py

```diff
@@ -39,7 +42,12 @@ def get_image_size(img: Any) -> List[int]:
 @torch.jit.unused
 def get_image_num_channels(img: Any) -> int:
     if _is_pil_image(img):
-        return len(img.getbands())
+        if hasattr(img, "getbands"):
+            channels = len(img.getbands())
+        else:
+            return img.channels  # accimage support
```

Comment:
This gives `AttributeError: channels`.

Reply:
I am running the command mentioned in the diff on my devserver.

Reply:
Using `conda install -c conda-forge accimage` from the installation instructions of accimage: https://github.com/pytorch/accimage#installation